Rakefile

```diff
@@ -1 +1,84 @@
+#  Copyright 2015 Puppet Community
```

Comment:
Actually that's a good point. In order to be able to hold copyright you need to be a legal entity or explicitly ask people to transfer the code into the ownership of the developers. That would become messy fast and leaves open questions like what happens when certain people leave etc.

Instead of going the full route of becoming an incorporated non-profit and the headaches that go with it we should probably see about joining http://sfconservancy.org. This would also allow me to transfer other titles like our domain name etc. onto them.